GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
Oh I see now, wouldn't this method break the intended life cycle for the Context? i.e. the context would only be wholly complete and correct after onInit(). It is fine in your case since you only use it to access the script manager which has been initialized by the constructor.